core/domain/summary_services.py

```diff
@@ -313,6 +406,31 @@ def get_displayable_exp_summary_dicts(exploration_summaries):
 
 
 def _get_displayable_collection_summary_dicts(collection_summaries):
+    """Gets a summary of collections in human readable form.
+
+    Args:
+        collection_summaries: list(CollectionSummary). List of collection
+        summary domain object
+
+    Return:
+        list(dict). A list of exploration summary(dict) in human readable form.
```

Comment:
Please modify this appropriately to be consistent with the pattern in previous functions.